### DISCUSSION TITLE
Xournal++ and XPS 13 2-in-1

### DISCUSSION THREAD

**Discussion by SimonApers** [xournalpp/xournalpp#3958] [Q&A]
Hi all,

I recently switched to an XPS 13 2-in-1 with Windows 10, and would like to use Xournal++ with the Dell Premium Active Pen (PN579X). Everything seems to work fine, except for one thing that really prevents me from using it. Xournal seems to put some sort of "threshold" so that very small movements of the pen get reduced to a point (which is very briefly surrounded by a circle). This behavior is exclusive to the pen (it does not show when I use the mouse or a Wacom tablet + stylus), and moreover it is not intrinsic to the pen (in Windows' OneNote there is no such threshold) so it seems to be a Xournal setting. I tried changing pretty much all settings, but couldn't find anything that solves this behavior.

I'd be very happy to hear any suggestions. For now I'm forced to use OneNote, but I really prefer Xournal++.

Thanks in advance!
Simon

**Comment by rolandlo** [comment]
There is a threshold (which is applied to both pen and mouse), but it is 0.3 pt = 0.0016 cm (with 100% zoom), which should be impossible to notice. Can you post a screenshot of your settings in the input system panel in menu `Edit > Preferences`? Maybe something is wrong there. A circle surrounding a point only appears for touch input as much as I have observed (on Windows 10). 
Also a screen recording might help us to understand the situation better.
